src/SearchCount/index.tsx

```diff
@@ -0,0 +1,24 @@
+import { connectStateResults, StateResultsProvided } from "react-instantsearch-core";
+
+/**
+  * Displays the total number of search hits, along with the query that was searched
+  */
+
+export const SearchCount = ({ searchResults }: Partial<StateResultsProvided>) => {
+  if (searchResults) {
+    const { nbHits, query } = searchResults;
+    return (
+     <>
+        <p className="search-pagination--count">
+          {`${nbHits} result${nbHits === 1 ? '' : 's'} for:`}
```

Comment:
Out of curiosity, how is it printing "0 result**s** for:" when this is checking if `nbHits === 1`? I searched for "288" to see 0 results:
![image](https://user-images.githubusercontent.com/29589689/131391003-665cde41-5f8c-4f08-b0d8-07c82aaa4299.png)

Reply:
The logic there is basically saying that if the `nbHits` value is anything **other than** `1`, it should show the `s` -- it doesn't really matter if the the value of `nbHits is less than or greater than `1`. Since we won't have negative or non-numeric values that's really the only situation where we'd need the word "result"; everywhere else we'd use "results"

Reply:
Ah oops sorry I'm not sure what I was thinking! Thanks for clarifying.